documents/src/pages/elements/datetime-picker.md

```diff
@@ -14,17 +14,28 @@ language_tabs: [javascript, typescript, css]
 ```
 ```css
 section {
-  height: 315px;
+  height: 400px;
   padding: 0 3px;
 }
+.container {
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+}
 [range][timepicker] {
   width: 400px;
 }
 ```
 ```html
 <section>
-  <ef-datetime-picker></ef-datetime-picker>
-  <ef-datetime-picker range duplex timepicker opened></ef-datetime-picker>
+  <div class="container">
+    <label for='datetime-picker'>Date of Birth</label>
+    <ef-datetime-picker id='datetime-picker'></ef-datetime-picker>
```

Comment:
preferably, `id` should be more semantic e.g. `date-of-birth`.

Reply:
Fixed